src/DesignMyNight/Elasticsearch/Connection.php

```diff
@@ -376,6 +378,41 @@ public function pretend(Closure $callback)
 
     }
 
+    /**
+     * Get the timeout for the entire Elasticsearch request
+     * @return float
+     */
+    public function getRequestTimeout(): float
+    {
+        return $this->requestTimeout;
+    }
+
+    /**
+     * Get the timeout for the entire Elasticsearch request
+     * @param float $requestTimeout
+     * @return self
+     */
+    public function setRequestTimeout(float $requestTimeout): self
```

Comment:
@willtj can you add the unit to the comments?

Reply:
Good thinking, done.